app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :user_id
+  has_many :comments
+  has_many :likes
```

Comment:
- Good job done so far, while the previous reviewer suggested that the customer methods for updating the `likes and posts` counter should be in their respective files, the method to display the `three_recent_posts` should be implemented here. Please fix this 🙏

Reply:
done